### PR TITLE
Update download.html to reflect the unit change

### PR DESCRIPTION
**Pull request by andresdelfino**
See #4293.

### CODE REVIEW

**Review by davidak — APPROVED** [review]

File: Doc/tools/templates/download.html [hunk 1]
```diff
@@ -13,7 +13,7 @@ <h1>Download Python {{ release }} Documentation</h1>
 
 <p>To download an archive containing all the documents for this version of
 Python in one of various formats, follow one of links in this table. The numbers
-in the table are the size of the download files in megabytes.</p>
+in the table are the size of the download files in mebibytes.</p>
```
Comment (vstinner) [inline Doc/tools/templates/download.html]: it's not a commonly used unit (I mean, everybody misuses "megabytes", but "mebibytes" term is rarely used), maybe add its value: megabytes (2\ :sup:`20` bytes).

I didn't test if it's formatted properly, :superscript: is for the 2^20 exponent.

Comment (andresdelfino) [reply Doc/tools/templates/download.html]: Alright :) I translated the idea to HTML.

**Review by vstinner — APPROVED** [review]
LGTM.

**Comment by vstinner** [comment]
@JulienPalard: Would you mind to review this PR?

**Comment by andresdelfino** [comment]
@JulienPalard could you take a look at this?

File: Doc/tools/templates/download.html [hunk 2]
```diff
@@ -13,7 +13,7 @@ <h1>Download Python {{ release }} Documentation</h1>
 
 <p>To download an archive containing all the documents for this version of
 Python in one of various formats, follow one of links in this table. The numbers
-in the table are the size of the download files in megabytes.</p>
+in the table are the size of the download files in megabytes (2<sup>20</sup> bytes).</p>
```
Comment (serhiy-storchaka) [inline Doc/tools/templates/download.html]:
```
1 megabyte = 10**6 bytes
1 mebibyte = 2**20 bytes
```

Comment (andresdelfino) [reply Doc/tools/templates/download.html]: @vstinner I followed your comment on https://github.com/python/cpython/pull/9639#discussion_r231696589

Perhaps your "megabytes" was a typo? I think mebibyte is the correct thing to say, by the way.

**Comment by vstinner** [comment]
An alternative is to remove the download sizes. I don't expect that release managers update this page as part of the release process:
https://www.python.org/dev/peps/pep-0101/

**Comment by andresdelfino** [comment]
> An alternative is to remove the download sizes. I don't expect that release managers update this page as part of the release process:
> https://www.python.org/dev/peps/pep-0101/

I also think that documentation size isn't that important nowadays, but perhaps there's a rationale we are missing here?

**Comment by davidak** [comment]
@andresdelfino there are regions here in germany where people only get very slow internet connections. So it makes a difference for them if it is 1 MB or 5 MB. But they might not be python developers...

**Comment by andresdelfino** [comment]
@davidak thanks for the insight!

@vstinner I'd keep the file sizes. Let me know how do you want to proceed on this one and I'll update the PR.

**Comment by andresdelfino** [comment]
@vstinner how should I proceed? Should I remove the file sizes or preserve them and fix the typo pointed by Serhiy?

**Comment by vstinner** [comment]
@andresdelfino: Honestly, I'm very confused by this issue. I don't understand why it takes months to fix such trivial issue :-) I skip my turn and let others handle it, sorry :-(

File: Doc/tools/templates/download.html (hunk 2 above)
Comment (andresdelfino) [reply Doc/tools/templates/download.html]: @serhiy-storchaka I changed the unit based on your comment.

**Comment by matrixise** [comment]
With the commit of @vstinner (see https://github.com/python/cpython/commit/8c663fd60ecba9c82aa4c404dbfb1aae69fe8553#diff-9c9d6c705ef5234b89019a8310a89d21), we don't need to specify the unit because it's explicitly indicated in the table.

Moreover, the commit removes the ambiguity with the unit.

The best thing to do is removing the complete line with the detailed unit.

I am going to close your PR and will fix the line.

Thank you a lot for your PR and hope to see your next contributions in the future.

**Comment by andresdelfino** [comment]
@matrixise thanks for the reply :)

**Comment by vstinner** [comment]
> The best thing to do is removing the complete line with the detailed unit. I am going to close your PR and will fix the line.

@matrixise: Would you mind to add a link to your fix here?

**Pull request closed without merging**